releasenotes/notes/deprecate-legacy-code-fe9db8a009b6f5d2.yaml

```diff
@@ -0,0 +1,43 @@
+---
+deprecations:
+  - |
+    Deprecated all code that was migrated to `qiskit_nature/second_q` according to issue:
+    https://github.com/Qiskit/qiskit-nature/issues/749
+
+    | Migration by package:
+    | The following table gives a more detailed breakdown.
```

Comment:
I'm not sure what formatting you attempted here. The output right now is as such:
![screenshot_1659627292](https://user-images.githubusercontent.com/21973473/182887842-00e8c70b-b08a-42bb-8779-6093c545279e.png)

The table looks nice imo, but maybe we can simply make this text as following:
```suggestion
    The following table provides a rough overview of the old and new package locations.
    In certain cases, single classes might have been treated differently.
    Be sure to check out the referenced issues or the migration guide for more details.
```

Where `"the migration guide"` will be added when the refactoring comes to and end, and will be linked to in its release note. We can then even go back into this reno, and make this a link, too :+1: